README.md

```diff
@@ -13,27 +13,30 @@ Beginning with the next version (0.9.0), **JWST requires Python 3.5 and above**.
 
 Installing
 ----------
-Releases are made on the **stable** branch. To install the latest release:
 
-    % git clone https://github.com/STScI-JWST/jwst.git
+To install the latest stable version of the library, we recommend using [conda](https://conda.io/docs/index.html) and
+the [astroconda](https://astroconda.readthedocs.io) channel:
 
-    % cd jwst
+    conda config --add channels http://ssb.stsci.edu/astroconda-dev
+    conda install jwst
 
-    % git checkout stable
+To install the development version of the repository, we recommend using the [astroconda](https://astroconda.readthedocs.io) channel
+to install the dependencies, and then installing from the github repository:
 
-    % python setup.py install
+    conda config --add channels http://ssb.stsci.edu/astroconda-dev
+    conda install jwst
+    git clone https://github.com/STScI-JWST/jwst.git
+    cd jwst
+    python setup.py develop
 
-The main development is on the **master** branch. To install the development version:
+If you will not need to update the code, then you can replace the last step with the following command:
 
-    % git clone https://github.com/STScI-JWST/jwst.git
+    python setup.py install
 
-    % cd jwst
+Once installed, the software can be updated to the lastest development version by running the following command inside the `jwst` 
+repository:
```

Comment:
Is it really correct to refer to a user's checked-out/cloned version of the repo as "the jwst repository"? When I read that, I think of the actual repository in github.